.gitignore

```diff
@@ -8,3 +8,5 @@ coverage
 .vs
 .vscode
 package-lock.json
+/.yarn/install-state.gz
+**/.pnp.*
```

Comment:
see https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

Reply:
Added their recommendations.